### PR TITLE
[stable/elastalert] Add support to custom SSL certificates

### PR DESCRIPTION
**Pull request by sneko**
**What this PR does / why we need it**:

Add support to custom SSL certificates

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

#7095 

**Special notes for your reviewer**:

@jertel I hope it will help 👍

### CODE REVIEW

**Comment by jertel** [comment]
@sneko Looks like the commit needs to be signed off. See this link for more info: https://www.helm.sh/blog/helm-dco/index.html

**Comment by sneko** [comment]
@jertel yes sorry, I'm not used to it. Seems good now 😃

**Comment by sneko** [comment]
I signed it!

**Comment by sneko** [comment]
@jertel read somewhere earlier in the day that "I signed it!" comment would relaunch the check...

Since `dco-labeler` is pending, what does it need to be done?

Thanks

File: stable/elastalert/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: ElastAlert is a simple framework for alerting on anomalies, spikes, or other patterns of interest from data in Elasticsearch.
 name: elastalert
-version: 0.5.0
+version: 0.5.1
```
Comment (davidkarlsen) [inline stable/elastalert/Chart.yaml]: new feature, so I'd say 0.6.0

Comment (sneko) [reply stable/elastalert/Chart.yaml]: @davidkarlsen I was hesitating while changing the version ^^... Just set it to 0.6.0 ;)

**Comment by jertel** [comment]
Thanks for your contribution @sneko, and for the chaart version bump. I noticed all recent PRs are lacking the "Contribution Allowed" label so the bot might be having trouble.

**Pull request merged**